README.md

```diff
@@ -64,7 +67,7 @@ For a future use you can omit the second step and run `up` (the third step) dire
 You need:
 - [Docker](https://docs.docker.com/docker-for-mac/install/) (to run NixOS in a container)
 - [XQuartz](https://www.xquartz.org/) (to share your screen with Docker)
-
+- Intel Mac device (ARM Mac's are not supported for now)
```

Comment:
Let's put this under the `### MacOS` directly without `-`. And maybe in _italic_.

Reply:
done